### ISSUE TITLE
Bug: cover property fails with wrong rule-ID

### ISSUE THREAD

**Issue by svenka3** [AsFigo/SVALint#57]
False alarm on below code, I didn't expect a failure on the rule:

[FUNC_MISSING_FAIL_ABLK] 1

```verilog
cover property (@(posedge clk) a |=> b);
```

Looks like this rule will need to filter cover/assume
